situp-plugins/common/src/main/java/com/amazon/situp/plugins/processor/StringProcessor.java

```diff
@@ -13,9 +13,11 @@
  * An simple String implementation of {@link Processor} which generates new Records with upper case content. The current
```

Comment:
nit: Write a readme for this to keep it consistent with rest of the plugins.

Reply:
Done